#!/usr/bin/env python3
PACKAGE = 'dutuuv_control'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('roll_p', double_t, 0, "...", default=0.0, min=0.0, max=100.0)
gen.add('roll_i', double_t, 0, "...", default=0.0, min=0.0, max=100.0)
gen.add('roll_d', double_t, 0, "...", default=0.0, min=-100.0, max=0.0)
gen.add('roll_i_clamp_max', double_t, 0, "...", default=0.0, min=-5.0, max=0.0)
gen.add('roll_i_clamp_min', double_t, 0, "...", default=0.0, min=-5.0, max=0.0)

exit(gen.generate(PACKAGE, 'balance_controller', 'BalanceController'))